machine-learning/house_price/regression-py.dig

```diff
@@ -28,7 +28,7 @@ _export:
     py>: tasks.FeatureSelector.run
     docker:
       # Note: This image is temporary. It might change in the near future.
```

Comment:
Do we still keep this note? I personally think it's helpful to provide a link to the [custom scripting documentation](https://support.treasuredata.com/hc/en-us/articles/360023947173-Python-Custom-Scripting-Tutorial) so that people can understand what this image is, how to customize it, ...

Reply:
Ah good place to link. Will add that